tests/src/test/scala/com/nvidia/spark/rapids/UrlFunctionsSuite.scala

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
Seems duplicated to ```url_test.py```. We can just keep ```url_test.py``` which can be run in parallel.

Reply:
Yes, it is just make it easy to test for now. Will remove it before it's ready for review.

Reply:
Removed.